src/app/clusters/level-control/level-control.cpp

```diff
@@ -958,6 +960,14 @@ static void moveHandler(app::CommandHandler * commandObj, const app::ConcreteCom
         goto send_default_response;
     }
 
+    if (!rate.IsNull() && (rate.Value() == 0))
```

Comment:
@cecille Even if merged already ... is this really the right place?

There is another second case for a comparable check for the defaultRate fallback which is way later in the code flow https://github.com/cecille/connectedhomeip/blob/97d435b8a498fc935f04f16743c657f2b660f04b/src/app/clusters/level-control/level-control.cpp#L1042 ... when I understood the code right now "rate===0" would not cause "withOnOff" cases to turn on/off with this logic, but a defaultRate === 0 would ... so that's a different behavior for "Kind of the same". Was that intended?

Reply:
@tennessee-google

Reply:
@Apollon77 I believe you are right. Both rate =0 or DefaultRate = 0 when rate is Null should lead to no action taken.

I think a small correction would be to evaluate the rate is Null and DefaultRate > 0 earlier in the handler

Reply:
issue here - https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/9125

Reply:
@jmartinez-silabs Thank you for your check. One question for detail: "No action taken at all"? or "Do nothing but process potential onoff logic"?
Case level  is 50% and onoff=off ... should be a call here leave onof===off or turn on as onoff cluster dependency logic would require it.
I ask because there are some other cases where - even without any action taken - the onoff dependecy logic is executed ... and this is also included in tests ...